cherryontech/jupiter-jumpers
# Update landing style and images

## Description

- Updated all the screenshot images
- Updated testimonials section
- Added the new cloud illustrations
- Implemented the two carousels
- Implemented the FAQ section

## Related Story Card

https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-51

## Type of Changes

Style and feature updates.

## Update Screenshots

### After
![Screenshot 2024-06-15 at 1 29 52 PM](https://github.com/cherryontech/Uncloud/assets/47403135/2a2d52c7-00f1-494c-9df2-e123bdb2e62b)
![Screenshot 2024-06-15 at 1 30 42 PM](https://github.com/cherryontech/Uncloud/assets/47403135/ce26d94f-46f2-42e3-9d81-05d6efa4a3d8)
![Screenshot 2024-06-15 at 1 30 51 PM](https://github.com/cherryontech/Uncloud/assets/47403135/6bb34973-6831-4af3-af93-efa5bf24957d)

## Testing Instructions

1. Try resizing the page a few times, test the first carousel by clicking the buttons. Ensure the transition doesn't look weird.
2. Try resizing the page a few times, and test the second carousel the same way.
3. Repeat the same (resizing, checking at different window sizes) for: FAQ, illustrations, and customer testimonials section.
